<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>539</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>22</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*#MainWindow{
background-image: url(:/icons/background(1).png);
}*/
/*** 标签 ***/
QLabel{
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 14px;
	font-weight: bold;
	color: grey;
	/*font: bold italic 18px &quot;Microsoft YaHei&quot;;*/
	border: 1px none grey;
}

/* 通用的蓝色按钮 */
QPushButton {
    background: white;
    color: grey;
    font-size: 12px;
    border: none;
	font-weight: bold;
	border-radius:10px;
}
QPushButton:pressed {
    background: white;
}

/* 单行编辑框 */
QLineEdit {
    background: rgba(204,204,204,0.3);
    color: #bbbbbb;
    font-size: 15px;
    padding-left: 2px;
    border: none;
}
QGroupBox
{
	border: 3px none gray;
	border-radius:10px;
	margin-top:4ex;
	font:bold 14px;
	color:grey; 
} 
QGroupBox::title
{
	subcontrol-origin:margin;
	subcontrol-position:top center;
	padding:1px;
}
QGroupBox::enabled{
	border: 3px none gray;
}
QGroupBox::!enabled {
	border: 1px none gray;
}
QLineEdit{
	background:rgba(0,0,0,0);
	color:white;
	font-weight:bold;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_weather">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>天气</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_reminderTitle">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:white;}</string>
    </property>
    <property name="text">
     <string>HERE IS YOUR REMINDERS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wenben">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>现在时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color:white;
}</string>
    </property>
    <property name="text">
     <string>——————————————————</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_city">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>南昌</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reminders">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>日程</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Week">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font:28px;	
	color:white;
}</string>
    </property>
    <property name="text">
     <string>FRIDAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Month_Day">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font:28px;	
	color:white;
}</string>
    </property>
    <property name="text">
     <string>APRIL 3RD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>271</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color:rgb(85, 170, 127);
	border-radius:10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Reminders">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color:white;
}</string>
    </property>
    <property name="text">
     <string>REMINDERS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cityWeather">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font:14px;	
	font-weight:bold;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>今天</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nowDateTime">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>（当前日期）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_help">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>查看</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_reminderLeft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>281</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea { 
	border: none;
	background-color: rgba(85, 170, 127,0); 
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>20</y>
       <width>291</width>
       <height>411</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_greenLeft">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>271</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	background-color:rgb(85, 170, 127);
	border-radius:10px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_weekLeft">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Betelguesse</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font-family: &quot;Betelguesse&quot;;
color:white;
font-size:23px;
}</string>
      </property>
      <property name="text">
       <string>FRIDAY</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_reminderClose">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>360</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>251</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	background:rgba(0,0,0,0);
}
QPushButton{
	color:white;
	font-size:20px;
}</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>251</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_calender">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>411</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_white">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>401</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background:white;
	border-radius:10px;
}
QPushButton{
	background:rgba(0,0,0,0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_left">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background:white;color:grey;font-weight:bold;}
QPushButton:pressed{background:white;}

QPushButton{
	background:rgba(0,0,0,0);
}</string>
     </property>
     <property name="text">
      <string>←</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QDateEdit
{
border : none;
background:rgba(0,0,0,0);
padding : 5px;
border-radius:10px;}
QDateEdit::up-arrow
{
border :none;
background-color : #00ff7f;

}
QDateEdit::down-arrow
{
border : none;

background-color : skyblue;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="displayFormat">
      <string>yyyy/M</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Year">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background:white;color:grey;font-weight:bold;font-size:18px;}
QPushButton:pressed{background:white;}

QPushButton{
	background:rgba(0,0,0,0);

}</string>
     </property>
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_right">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background:white;color:grey;font-weight:bold;}
QPushButton:pressed{background:white;}

QPushButton{
	background:rgba(0,0,0,0);
}</string>
     </property>
     <property name="text">
      <string>→</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_Buttons">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>341</width>
       <height>331</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size:13px;
	font-weight:bold;
	height:41px;
	background:rgba(0,0,0,0);
}
QLabel{
	height:39px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_16">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButton_Week_5">
        <property name="text">
         <string>周五</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="pushButton_27">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pushButton_36">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton_Week_4">
        <property name="text">
         <string>周四</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_Week_2">
        <property name="text">
         <string>周二</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_15">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="pushButton_Week_6">
        <property name="text">
         <string>周六</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="pushButton_38">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="pushButton_12">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="pushButton_Week_7">
        <property name="text">
         <string>周日</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_9">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="pushButton_30">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pushButton_31">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButton_18">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_8">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_22">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_Week_1">
        <property name="text">
         <string>周一</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton_29">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="pushButton_40">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="pushButton_21">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_17">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="pushButton_39">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="pushButton_25">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_24">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="pushButton_13">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_Week_3">
        <property name="text">
         <string>周三</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QPushButton" name="pushButton_28">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="pushButton_32">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="pushButton_37">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QPushButton" name="pushButton_42">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="pushButton_33">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QPushButton" name="pushButton_34">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="pushButton_20">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_11">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QPushButton" name="pushButton_41">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_10">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="pushButton_26">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QPushButton" name="pushButton_35">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="pushButton_14">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_23">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="pushButton_19">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_openTheWholeYear">
     <property name="geometry">
      <rect>
       <x>364</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>⬇</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nowDateTime_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>400</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>物联网工程2023级1班-张祖宁</string>
     </property>
    </widget>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButton_weather</zorder>
   <zorder>label_reminderTitle</zorder>
   <zorder>label_wenben</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit_city</zorder>
   <zorder>pushButton_reminders</zorder>
   <zorder>label_Week</zorder>
   <zorder>pushButton_exit</zorder>
   <zorder>label_Month_Day</zorder>
   <zorder>label_Reminders</zorder>
   <zorder>label_cityWeather</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_nowDateTime</zorder>
   <zorder>pushButton_help</zorder>
   <zorder>groupBox_calender</zorder>
   <zorder>groupBox_reminderLeft</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reminderClose</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_reminderLeft</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>drawCalendar_slot(QDate)</slot>
  <slot>slot1()</slot>
 </slots>
</ui>
